.github/workflows/continuous-integration.yaml

```diff
@@ -44,24 +44,17 @@ jobs:
         distribution: 'zulu'
         java-version: '8'
 
-    - name: Cache dependencies
-      uses: actions/cache@v2
-      with:
-        path: |
-          jetty-runner.jar
-          road-traffic-rest-2.0.0.war
-        key: road-traffic-dependencies
-
     - name: Start road-traffic example
       run: |
+        gafferVersion=`sed -n '/^__version__\ =\ / {s///p;q;}' src/__init__.py | tr -d '"'`
```

Comment:
Technically you could do this all with sed by matching and saving the version e.g.

```bash
sed 's/^__version__.*\([0-9]\{1,\}\.[0-9]\{1,\}\.[0-9]\{1,\}\).*/\1/'
```

or just matching whats between the quotes e.g.

```bash
sed 's/^__version__.*"\(.*\)"/\1/'
```

Reply:
Agree that sed regex group matching would be better. I think matching between the quotes would be best as the version will never contain a quote and we are not trying to validate that it's valid (is done elsewhere by this point).

The command to use is `sed -n 's/^__version__.*"\(.*\)"$/\1/p' src/__init__.py`, as we only want to 'print' the matched version and not subsequent lines where there's no match.